### ISSUE TITLE
possible settings options

### ISSUE THREAD

**Issue by ondrobaco** [ivan-lednev/better-search-views#32]
Hi, thanks for this fantastic plugin! 
I'd like to suggest a useful setting option. 
I use a lot of embed search, which pulls from a long document and create smaller documents that consist from the search results.
The embed search views however does not show the whole paragraph or whole list of items. 
An option is setting where a user can search the granularity of the search would be good idea....
example: search and display everything 
below h1 heading (displays everything up to the next h1 heading)
below h2 heading (view with everything up to the next h2 heading)
and so on.

**Comment by jemorriso** [comment]
I came with the exact same use case and found this issue. It would be fantastic to see the search results include the entire section below a header, not just the first paragraph, at least as an option in settings.

**Comment by ondrobaco** [comment]
yes, even a plugin that can extract between a specified string would be great. Less convenient than headers but I wouldn't mind to add a string in my "motherload" to be able to extract entire sections... Perhaps an idea for a new plugin?

**Comment by codey-badger** [comment]
Agree. Did you find a workaround?

**Comment by ondrobaco** [comment]
> Agree. Did you find a workaround?

kind of. If you tag each section with a tag/s, you could ask AI to extract blocks of text that contain given tag and compile a document out of them.
